Sources/Brave/Frontend/Settings/FilterListAddURLView.swift

```diff
@@ -0,0 +1,111 @@
+// Copyright 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import Strings
+import DesignSystem
+import BraveUI
+
+struct FilterListAddURLView: View {
+  @ObservedObject private var customFilterListStorage = CustomFilterListStorage.shared
+  @Environment(\.presentationMode) @Binding private var presentationMode
+  @State private var newURLInput: String = ""
+  @State private var errorMessage: String?
+  
+  private var textField: some View {
+    TextField(Strings.filterListsEnterFilterListURL, text: $newURLInput)
+      .onChange(of: newURLInput) { newValue in
+        errorMessage = nil
+      }
+      .keyboardType(.URL)
+      .textContentType(.URL)
+      .autocapitalization(.none)
+      .autocorrectionDisabled()
+  }
+  
+  var body: some View {
+    NavigationView {
+      List {
+        Section(content: {
+          VStack(alignment: .leading) {
+            textField
+              .submitLabel(SubmitLabel.done)
+            
+            if let errorMessage = errorMessage {
+              Text(errorMessage)
+                .font(.caption)
+                .foregroundColor(.red)
+            }
+          }
+        }, header: {
+          Text(Strings.customFilterListURL)
+        }, footer: {
+          VStack(alignment: .leading, spacing: 8) {
+            Text(Strings.addCustomFilterListDescription)
+            Text(LocalizedStringKey(Strings.addCustomFilterListWarning))
+          }.padding(.top, 16)
```

Comment:
16 pts of padding is too much, lets reduce this to 8

Reply:
Ok no problem, but I'm just followed the design I was given.